packages/flutter/test/material/buttons_test.dart

```diff
@@ -310,7 +310,27 @@ void main() {
     );
   }, skip: true);
 
-  testWidgets('Disabled MaterialButton has same semantic size as enabled and exposes disabled semantics', (WidgetTester tester) async {
+  testWidgets('Disabled MaterialButton has correct default text color', (WidgetTester tester) async {
+    const String testText = 'Disabled';
+    const Widget buttonWidget = Directionality(
+      textDirection: TextDirection.ltr,
+      child: Material(
+        child: Center(
+          child: MaterialButton(
+            onPressed: null,
+            child: Text(testText), // button is disabled
+          ),
+        ),
+      ),
+    );
+
+    await tester.pumpWidget(buttonWidget);
+
+    final RichText text = tester.widget(find.byType(RichText));
```

Comment:
Wouldn't `tester.widget<Text>(find.text(testText)).style` work here? The test probably doesn't need to depend on the fact that the Text widget is defined in terms of RichText.

Reply:
No. You have to look for the RichText widget. The `style` on Text widgets is `null` when there's no theme. But it does draw. Just with private API to the RichText.